### DISCUSSION TITLE
突然发现xmoj的老讨论区可以用

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#389] [General]
https://www.xmoj.tech/discuss3/thread.php?tid=211 

请关闭脚本

**Comment by boomzero** [comment]
<img width="1648" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/a6951fef-f062-4075-bf9a-692f1e0b3b08">

**Comment by boomzero** [comment]
所以feature request, 使用xmoj原有讨论区

**Comment by boomzero** [comment]
trick: 去https://www.xmoj.tech/discuss3/discuss.php?pid=1250&cid=1930 发帖, 不要在mainboard发

**Comment by PythonSmall-Q** [comment]
一票否决，不换

**Comment by boomzero** [comment]
Why not? At least write a synchroniser

**Comment by boomzero** [comment]
Apparently my email application wants me to follow up on this conversation, so any news?

**Reply by PythonSmall-Q** [reply]
nope，你用的是什么**email application

**Reply by boomzero** [reply]
@PythonSmall-Q 
<img width="396" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/ca837959-4ce1-484b-b8a7-990003e0dbf4">

**Reply by PythonSmall-Q** [reply]
6 挺智障的
